zuul.d/ansible-cloud-jobs.yaml

```diff
@@ -40,3 +40,12 @@
     vars:
       ansible_test_python: 3.6
       ansible_test_integration_targets: zuul/vmware/vcenter_only/
+
+- job:
+    name: ansible-test-cloud-integration-vcenter_1esxi-python36
+    parent: ansible-test-cloud-integration-vcenter
```

Comment:
okay, i think I see the issue here. You've opted to parent to ansible-test-cloud-integration-vcenter over creating an new esxi specific one, which also include vcenter.

The part I need to understand, can esxi be run without vcenter? My gut says, we should likely create ansible-cloud-esxi-appliance. Then, this job parent to both ansible-cloud-vcenter-appliance and ansible-cloud-esxi-appliance.

Stacking it this way, kinda works, but not means that esxi things are in the vcenter playbooks. Which makes review / flow a little messy

Reply:
> okay, i think I see the issue here. You've opted to parent to ansible-test-cloud-integration-vcenter over creating an new esxi specific one, which also include vcenter.
> 
> The part I need to understand, can esxi be run without vcenter? My gut says, we should likely create ansible-cloud-esxi-appliance. Then, this job parent to both ansible-cloud-vcenter-appliance and ansible-cloud-esxi-appliance.
> 
> Stacking it this way, kinda works, but not means that esxi things are in the vcenter playbooks. Which makes review / flow a little messy

We can technically start a single ESXi instance and so in this case a `ansible-cloud-esxi-appliance` would make sense. But none of the tests relies on this kind of architecture. We always assume a vcenter is also available. So I took the initiative to just maintain on single playbook. Also, can a job have multiple inheritance?